File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.logging.Logger;
import java.util.regex.Pattern;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String input = br.readLine();

        String[] croatia = {"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};
        for (int i = 0; i < croatia.length; i++) {
            input = input.replaceAll(croatia[i], "1");
        }

        System.out.println(input.length());
    }
}

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.js
const fs = require('fs');
let input = fs.readFileSync('./dev/stdin').toString().trim();

const croatia = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z='];
for (let i = 0; i < croatia.length; i++) {
    input = input.replaceAll(croatia[i], '1');
}
console.log(input.length);
